server/db/index.js

```diff
@@ -1,22 +1,22 @@
 //DB
-const db = require('./db');
+const db = require("./db");
 //MODELS
-const Cart = require('./models/Cart');
-const Guest = require('./models/Guest');
-const Product = require('./models/Product');
-const Review = require('./models/Review');
-const Transaction = require('./models/Transaction');
-const User = require('./models/User');
+const Cart = require("./models/Cart");

```

Comment:
Though I have uncertainty at the moment. I feel confident we don't want to comment out these associations